tasks/size_contracts.js

```diff
@@ -13,6 +13,23 @@ const formatSize = function (size) {
   return (size / 1024).toFixed(3);
 };
 
+const ansiRegex = function ({ onlyFirst = false } = {}) {
+  const pattern = [
+    '[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]+)*|[a-zA-Z\\d]+(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]*)*)?\\u0007)',
+    '(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PR-TZcf-nq-uy=><~]))'
+  ].join('|');
+
+  return new RegExp(pattern, onlyFirst ? undefined : 'g');
+}
+
+const stripAnsi = function (string) {
+  if (typeof string !== 'string') {
+    throw new TypeError(`Expected a \`string\`, got \`${typeof string}\``);
+  }
```

Comment:
This type checking is probably not necessary since we'll only be calling this from one place, and always with a string.  In the future I intend to convert my HH plugins to TS anyway.

Reply:
Sure, will get it removed